wavelength-plugins/misc.js

```diff
@@ -484,6 +484,30 @@ exports.commands = {
 		if (!this.runBroadcast()) return false;
 		if (!target || target === 'help') return this.parse('/help wlssb');
 		let targetData = getMonData(toId(target));
+		if (toId(target) === toId('km')) return this.sendReplyBox(getMonData('krakenmare'));
+		if (toId(target) === toId('callie')) return this.sendReplyBox(getMonData('callieagent1'));
+		if (toId(target) === toId('c7')) return this.sendReplyBox(getMonData('c733937123'));
+		if (toId(target) === toId('deso')) return this.sendReplyBox(getMonData('desokoro'));
+		if (toId(target) === toId('des')) return this.sendReplyBox(getMonData('desokoro'));
+		if (toId(target) === toId('mos')) return this.sendReplyBox(getMonData('mosmero'));
+		if (toId(target) === toId('electricz')) return this.sendReplyBox(getMonData('mosmero'));
+		if (toId(target) === toId('prince')) return this.sendReplyBox(getMonData('wavelengthprince'));
+		if (toId(target) === toId('hh')) return this.sendReplyBox(getMonData('hoeenhero'));
+		if (toId(target) === toId('hiro')) return this.sendReplyBox(getMonData('hiroz'));
+		if (toId(target) === toId('hoeen')) return this.sendReplyBox(getMonData('hoeenhero'));
+		if (toId(target) === toId('[]')) return this.sendReplyBox(getMonData('arrays'));
+		if (toId(target) === toId('mech')) return this.sendReplyBox(getMonData('iSteelX'));
+		if (toId(target) === toId('mechsteelix')) return this.sendReplyBox(getMonData('iSteelX'));
+		if (toId(target) === toId('cubs')) return this.sendReplyBox(getMonData('cubsfan38'));
+		if (toId(target) === toId('bunnery')) return this.sendReplyBox(getMonData('bunnery5'));
+		if (toId(target) === toId('rittz')) return this.sendReplyBox(getMonData('therittz'));
+		if (toId(target) === toId('stabby')) return this.sendReplyBox(getMonData('stabbythekrabby'));
+		if (toId(target) === toId('stk')) return this.sendReplyBox(getMonData('stabbythekrabby'));
+		if (toId(target) === toId('twb')) return this.sendReplyBox(getMonData('tidalwavebot'));
+		if (toId(target) === toId('tidal')) return this.sendReplyBox(getMonData('tidalwavebot'));
+		if (toId(target) === toId('lyc')) return this.sendReplyBox(getMonData('lycaniumz'));
+		if (toId(target) === toId('lycan')) return this.sendReplyBox(getMonData('lycaniumz'));
+		if (toId(target) === toId('vxn')) return this.sendReplyBox(getMonData('lycaniumz'));
```

Comment:
this can be simplified a bit
```javascript
if (['lyc', 'lycan', 'vxn'].includes(toId(target))) return this.sendReplyBox(getMonData('lycaniumz'));
```
do the same for the other aliases

Reply:
Yeah, that was mb, I never cleaned up, ty for the clean up @Lycanium-Z